File: trunk/net.tenteCsharp.templatesProject/src_code/heaterManagement/ITermometerGUINotify.java
package heaterManagement;
import heaterManagement.HeaterManagement.*;

public interface ITermometerGUINotify{
	
	public void newTemparature(String thermometerId,float value);	
	public void newOutsideTemperature(String termometerId,float value);	
	public String getTermometerId();

}

File: Elio/pseudoCodeGeneratorElio/src-gen/heaterManagement/ITermometerGUINotify.cs
using System;
using System.Collections;
using System.Text;

namespace SmartHome
{
	public interface ITermometerGUINotify
	{
		void newTemparature(String thermometerId, float value);
		void newOutsideTemperature(String termometerId, float value);
		String getTermometerId();

	}
}
